src/components/Button.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { pallete } from "../misc/pallete";
+
+export default function Button({ size, text, onClick, clicked = false }) {
+  return size === "small" ? (
+    <SmallBtn clicked={clicked} onClick={onClick}>
+      {text}
+    </SmallBtn>
+  ) : (
+    <MediumBtn clicked={clicked} onClick={onClick}>
+      {text}
+    </MediumBtn>
+  );
+}
+Button.propTypes = {
+  size: PropTypes.string,
```

Comment:
jeśli skorzystasz ze wskazówki wyżej (stosowanie obiektu i właściwości obiektu zamiast stringa) to wówczas tutaj możesz skorzystać z 'PropTypes.oneOf()' gdzie zdefiniujesz możliwe wartości. Możesz skorzystać z funkcji Object.values(buttonSize) aby wyciągnąć możliwe wartości i zawrzeć je tutaj w tablicy (bo taką składnię ma 'oneOf'),
czyli będzie coś w typie:

```
size: PropTypes.oneOf([Object.values(buttonSize)])
```

Reply:
dodatkowo, wymagane propsy powinny mieć dopisek 'isRequired'. np
  requiredFunc: PropTypes.func.isRequired,